package.json

```diff
@@ -23,6 +23,7 @@
   "license": "ISC",
   "scripts": {
     "rebel:re": "npm run rebel && ./rebel.sh examples/reason-project",
+    "rebel:rs": "npm run rebel && ./rebel.sh examples/recursive-src",
```

Comment:
Can you turn on travis CI for this repo? It will make sure that these examples compile. Probably build times will be a bit long. But it's fine.